### ISSUE TITLE
[Bug]: Label of checkbox is not a label

### ISSUE THREAD

**Issue by vogelor** [tasteui/tasteui#128]
### What Happened

When i create a checkbox (or when i look at your own demo / documentation) the label of the checkbox is NOT the label. it is outside the label-Tag in html. This means if i click at the label of a checkboc, the checkbox get not checked.

### How to Reproduce

Open your documentation at 
https://tallstackui.com/docs/form/checkbox
and click at a label of the checkbox. You can see, that the checkbox didn't get checked.

### Laravel Version

its your doc-page

### Livewire Version

its your doc-page

### TallStackUi Version

its your doc-page

### PHP Version

its your doc-page

### Operation System

Linux

### Notes

this is your html-code:

```html
                <label class="relative inline-flex cursor-pointer items-center">
            <input type="checkbox" class="form-checkbox rounded transition ease-in-out duration-100 border-secondary-200 w-5 h-5 text-primary-500 focus:ring-primary-500 dark:ring-offset-dark-900" checked="">
        </label>
                <p class="text-sm font-medium text-gray-700 dark:text-dark-400 ml-2">            Primary        </p>
```
The text has to be INSIDE the <label></label>

**Comment by devajmeireles** [comment]
Hey, @vogelor ! Thanks for your report.

I'm checking this right now.

**Comment by devajmeireles** [comment]
Thanks for your report, @vogelor .

This is a bug in the malformation of the input's HTML. We will address a fix today.

**Comment by devajmeireles** [comment]
@vogelor , thanks.

This and #129 were addressed at #130 also with the addition of `xs` variation.

Due to our policy, I'm closing this issue. Feel free to open a new issue whenever necessary.
